### DISCUSSION TITLE
Add Fallback to Website

### DISCUSSION THREAD

**Discussion by agravity-philipp** [rocksdanister/lively#1450] [Ideas]
Currently I use a picture from an URL as background. Sometimes I am in a foreign wifi network where I have to connect first. 
So the background looks totally broken. (Have to open Lively to switch back and forth to update the background after I have connected).

Is it possible to add a fallback picture or caching to see what was loaded the last time?

**Comment by rocksdanister** [comment]
Is the picture dynamically changing? is that why you are using url? ideally loading the picture directly will be better https://github.com/rocksdanister/lively/issues/1412

>Is it possible to add a fallback picture or caching to see what was loaded the last time?

Right now the browser is set to use memory cache, turning on disk cache did not help (I think its because I am listening for the loading error and showing custom error message.)

**Reply by agravity-philipp** [reply]
Hi! This is a logo on a fixed URL which seasonally changes. (Halloween, Pride, Easter, etc.) . So a refresh on startup is fine (so I know which variant of the logo is currently published) - But when you startup without Internet you see the error response as background which is not so good. Having a static image as fallback would be good.

![image](https://user-images.githubusercontent.com/81114510/200090709-26a09dda-341c-42d6-97c1-7ee6dab6c02f.png)

**Comment by rocksdanister** [comment]
Moved to issue https://github.com/rocksdanister/lively/issues/1454
